src/plone/restapi/tests/test_atcontent_serializer.py

```diff
@@ -193,3 +194,119 @@ def test_get_is_folderish_in_folder(self):
         obj = serializer()
         self.assertIn("is_folderish", obj)
         self.assertEqual(True, obj["is_folderish"])
+
+    def test_nextprev_alone(self):
```

Comment:
test_next_previous_on_empty_folder

Reply:
the folder is not empty, have just one document where I try to look for previous and next items